pkg/cmd/run.go

```diff
@@ -104,7 +104,7 @@ func RunMicroshift(cfg *config.MicroshiftConfig, flags *pflag.FlagSet) error {
 		util.Must(m.AddService(controllers.NewKubeScheduler(cfg)))
 		util.Must(m.AddService(controllers.NewKubeControllerManager(cfg)))
 		util.Must(m.AddService(controllers.NewOpenShiftCRDManager(cfg)))
-		util.Must(m.AddService(controllers.NewOpenShiftControllerManager(cfg)))
+		util.Must(m.AddService(controllers.NewRouteControllerManager(cfg)))
```

Comment:
are the commands running with admin privileges? Is RBAC going to be introduced in some way for all these components?

Reply:
I think that's part of the work @stlaz is doing, isn't it? See #898 for example.